docker/README.md

```diff
@@ -62,12 +62,90 @@ docker-compose -f docker-compose-mysql.yml down
 
 Please note that SQL support is still in active developement and it is not production ready yet.
 
-Quickstart for production
+One host production setup with docker-compose
 =========================
-In a typical production setting, dependencies (cassandra / statsd server) are
-managed / started independently of the cadence-server. To use the container in
-a production setting, use the following command:
+For a simple production setup, start with our **docker-compose-production-example.yml** for a single host Cadence cluster as described in the following steps.
 
+#### 1. Modify the database mounting path in the example YAML file
```

Comment:
This heading style doesn't match the rest of the file. Can you change the other headings to use the `# Heading` style?

Reply:
Done